local-volume/README.md

```diff
@@ -22,9 +22,20 @@ directories by creating and cleaning up PersistentVolumes for each volume.
 * The static provisioner only discovers mount points.  For directory-based local
   volumes, they must be bind-mounted into the discovery directories.
 
+## Version Compatibility
+
+Recommended provisioner versions with Kubernetes versions
+
+| Provisioner version | K8s version   | Reason                    |
+| ------------------- | ------------- | ------------------------- |
+| 2.1.0               | 1.10          | Beta API default, block   |
+| 2.0.0               | 1.8, 1.9      | Mount propagation         |
```

Comment:
Assuming the user wants to run 2.0.0 on K8S 1.8, it would be useful to put links in this table to those docs. For example, for 2.0.0, they should go to the tag https://github.com/kubernetes-incubator/external-storage/tree/local-volume-provisioner-v2.0.0/local-volume. I know this is available in changelog too, but that is a bit buried in there.

Reply:
done